web/src/components/PartProfile/PartProfile.js

```diff
@@ -88,9 +88,11 @@ const PartProfile = ({
             className="mt-6 ml-auto hover:shadow-lg bg-gradient-to-r from-transparent to-indigo-100"
             shouldAnimateHover
             iconName="chevron-down"
-            onClick={() => {}}
+            onClick={() => {
+              document.getElementById('section').scrollIntoView()
+            }}
           >
-            Comments 11
+            {userPart.Part.Comment.length}
```

Comment:
Could we change this to `Comments {userPart.Part.Comment.length}`. Otherwise it's not clear what the button is for.

Reply:
okay